README.md

```diff
@@ -13,121 +13,9 @@ The [Okta developer site](https://developer.okta.com) serves Okta's API document
 - [Guides](https://developer.okta.com/guides/)
 - [Developer Blog](https://developer.okta.com/blog/) (not published from this repo, see [okta/okta.github.io](https://github.com/okta/okta.github.io))
 
-## Getting Started
+## Building the site on your machine or contributing to the site 
+To build the site on your machine and/or to contribute content to the site, [view the instructions for Getting set up](https://github.com/okta/okta-developer-docs/wiki/Getting-set-up).  
 
-Okta's developer documentation (this repo) is built using the [VuePress](https://vuepress.vuejs.org/) site generator.
-There are currently 2 parts to the site, the content and the theming/plugins.
-
-### Requirements
-
-- [Node](https://nodejs.org/en/download/): 8+
-- [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable): 1.15+
-
-We recommend using one of the package manager options for installation as specified in the installation sites.
-
-If you want to run `yarn build`, you must have a node version of 13+
-
-Before getting started, open a terminal window and make sure these commands work:
-
-```sh
-node --version
-
-yarn --version
-```
-
-### Local setup
-
-1. Depending on your permissions, clone or fork the repo (if you fork the repo, be sure to [Allow edits from maintainers](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork)).
-2. Install the dependencies with `yarn`:
-
-```sh
-cd okta-developer-docs
-
-yarn install
-```
-
-This will install everything you need to build the documentation on your machine.
-
-### Previewing the site
-
-With the above steps completed, you can start a preview server by running this command inside the cloned directory:
-
-```sh
-yarn dev
-```
-
-This starts a preview server on your machine, and watches all files for changes. Open <http://localhost:8080/docs/> to view the documentation.
-
- > Note: if you try to visit the root, you will get a 404 page.  You must visit a path corresponding to a [directory under `vuepress-site`](https://github.com/okta/okta-developer-docs/tree/master/packages/%40okta/vuepress-site), like `/docs/`.
-
-The preview server supports hot reloading. Once the server is running on your machine, any changes you make to Markdown content will appear automatically in your browser within a few seconds. Note that changes to page frontmatter or site configuration require you to stop and start the preview server.
-
-## Troubleshooting
```

Comment:
This section heading needs to remain, and to link to the Troubleshooting page in the GH wiki.

Reply:
Fixed.